packages/gasket-nextjs/src/index.d.ts

```diff
@@ -0,0 +1,46 @@
+import { Document as NextDocument } from 'next';
+import { GasketData } from '@gasket/data';
+
+/**
+ * Renders a script tag with JSON gasketData
+ *
+ * @param {object} props - Props
```

Comment:
I'm curious whether you can omit the jsdocs types in these `.d.ts` files since it's redundant with the actual types in the TS code.

Reply:
Yea, I was wondering about that myself. The JSDocs do provide additional details for tooltips, etc, for example:
![Screen Shot 2022-03-25 at 2 34 40 PM](https://user-images.githubusercontent.com/82775/160204231-7c2940af-0b67-44c1-a93a-132d62a6b17c.png)
.